File: Runtime/TSTTransitionGroup.vb
'*********************************************************************************************
' ST-Sim: A SyncroSim Module for the ST-Sim State-and-Transition Model.
'
' Copyright © 2007-2017 Apex Resource Management Solution Ltd. (ApexRMS). All rights reserved.
'
'*********************************************************************************************

Friend Class TstTransitionGroup

    Private m_GroupId As Integer

    Public Sub New(ByVal groupId As Integer)
        Me.m_GroupId = groupId
    End Sub

    Public ReadOnly Property GroupId As Integer
        Get
            Return Me.m_GroupId
        End Get
    End Property

End Class

File: Runtime/TSTTransitionGroup.cs
// A SyncroSim Package for developing state-and-transition simulation models using ST-Sim.
// Copyright © 2007-2018 Apex Resource Management Solution Ltd. (ApexRMS). All rights reserved.

namespace SyncroSim.STSim
{
    internal class TstTransitionGroup
    {
        private int m_GroupId;

        public TstTransitionGroup(int groupId)
        {
            this.m_GroupId = groupId;
        }

        public int GroupId
        {
            get
            {
                return this.m_GroupId;
            }
        }
    }
}
